nucliadb_node/src/env.rs

```diff
@@ -0,0 +1,337 @@
+// Copyright (C) 2021 Bosutech XXI S.L.
+//
+// nucliadb is offered under the AGPL v3.0 and as commercial software.
+// For commercial licensing, contact us at info@nuclia.com.
+//
+// AGPL:
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+//
+use std::env;
+use std::net::{IpAddr, SocketAddr, ToSocketAddrs};
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::time::Duration;
+
+use nucliadb_core::tracing::*;
+
+use crate::utils::{parse_log_level, reliable_lookup_host};
+
+const SENTRY_PROD: &str = "prod";
```

Comment:
Should not be defined

Reply:
Im not entirely sure what you suggest, we are using that variable on telemetry and since this code was not made by me I am a bit lost. Are you suggesting this should come from a env var?

Reply:
Explained on next comment

Reply:
Sorry I understood that is the environment to send to sentry